### DISCUSSION TITLE
Download archived site pages

### DISCUSSION THREAD

**Discussion by Ygarr** [setvisible/DownZemAll#198] [Ideas]
Please add archived site pages in htz and/or rdf format

**Comment by setvisible** [comment]
Thanks for this interesting proposition.
I didn't know nothing about these formats, they seem intended for web scrapping. It could be an option for the future to scrap the web (meaning, downloading the content data, text, etc), but currently DZA is designed as a media file downloader.
I couldn't implement it soon, as my time availabities are limited. Somebody else could do it maybe. Otherwise, let's keep this option for later.
Regards,
